script/client

```diff
@@ -230,11 +223,7 @@ sub handle_result {
 # prepend the API-base if the specified path is relative
 sub prepend_api_base {
     my $path = shift;
-
-    if ($path !~ m/^\//) {
-        $path = join('/', $options{apibase}, $path);
-    }
-    return $path;
+    return join('/', $options{apibase}, $path) if $path !~ m/^\//;
```

Comment:
This will return the empty string if `$path =~ m/^\//`.

Reply:
done